### ISSUE TITLE
Error about missing inner_block assign when attempting to render a form in a component

### ISSUE THREAD

**Issue by jeffometer** [mhanberg/temple#136]
This is the bug I get: `** (ArgumentError) assign @inner_block not available in eex template.`

This is the code:

live/page_live.ex:
```elixir
      render do
        c Modal, on_cancel: "cancel" do
          "Form here:"

          c LiveFormFor,
            form:
              form_for(@changeset, "#",
                id: "create-meeting-id",
                data: [role: "standup"],
                phx_submit: "create-meeting",
                phx_hook: "ModalFocusSandbox"
              ) do
            slot :form, %{form: form} do
#etc...
```

components/modal.ex
```elixir
defmodule App.Component.Modal do
  import Temple.Component

  render do
    div class: "opacity-80 bg-black bg-opacity-80 inset-0 absolute",
        "phx-click": @on_cancel do
      slot :default
    end
  end
end

```

components/form.ex
```elixir
defmodule App.Component.LiveFormFor do
  import Temple.Component

  render do
    @form

    slot :form, form: @form

    "</form>"
  end
end
```

Now if I render the form _outside_ of the modal component everything works fine.  Similarly, if I render another modal component inside the modal component everything works fine.  If I remove the `slot :form ...` from the form component I don't get the error.

Do you know why this is happening or what it means?  Do you need any additional information?  This is with 0.6.0-rc.1 with Elixir 1.11.4.

Thank you.

**Comment by jeffometer** [comment]
Upon more exploration, these seems to specifically be a problem when using named slots.  `slot :default` works correctly, but `slot :named` wont.

**Comment by mhanberg** [comment]
I think this might have been fixed in 0.6.1 https://github.com/mhanberg/temple/issues/126

**Comment by jeffometer** [comment]
Yes!  You are correct, thank you!  Sorry I didn't notice the bug fix previously.
